draft-ietf-jsonpath-base.md

```diff
@@ -806,20 +796,22 @@ in the table below:
 | \\uXXXX            | U+XXXX              | unicode character           |
 {: title="Escape Sequence Replacements" cols="c c"}
 
-The `index-selector` applied with a `quoted-member-name` to an object
+A `dotted-member-name` string is converted to a member name by removing
+the initial dot.
+
+The name //PICKER// applied to an object
 selects the node of the corresponding member value from it, if and only if that object has a member with that name.
```

Comment:
//PICKER// descriptions need to avoid saying that a //PICKER// can _select_ a value, as this confuses //PICKERS// and selectors.